docker-compose.yaml

```diff
@@ -54,5 +54,13 @@ services:
       - '/usr/bin/env'
       - 'python3'
       - '/home/kernelci/pipeline/kcidb.py'
+
+  test_report:
+    <<: *base-service
+    container_name: 'test_report'
+    command:
+      - '/usr/bin/env'
+      - 'python3'
+      - '/home/kernelci/pipeline/test_report.py'
       - '--settings=/home/kernelci/config/kernelci.conf'
       - 'run'
```

Comment:
This is breaking the `kcidb` service definition, the last 2 lines are being removed from it.

Reply:
Fixed.